Library/Homebrew/cask/artifact/moved.rb

```diff
@@ -92,10 +92,12 @@ def move(adopt: false, force: false, verbose: false, predecessor: nil, reinstall
 
         if target.directory?
           if target.writable?
-            source.children.each { |child| FileUtils.move(child, target + child.basename) }
+            source.children.each { |child| FileUtils.move(child, target/child.basename) }
           else
-            command.run!("/bin/cp", args: ["-pR", "#{source}/*", "#{source}/.*", "#{target}/"],
-                                    sudo: true)
+            source.children.each do |child|
+              command.run!("/bin/cp", args: ["-pR", child, target/child.basename],
+                                      sudo: true)
+            end
```

Comment:
Does this work?
```suggestion
            command.run!("/bin/cp", args: ["-pR", *source.children, target],
                                    sudo: true)
```

Reply:
That does appear to work.

Reply:
Seems simpler, no? We'd also have to shell out fewer times, which makes this faster.